Gemfile.lock

```diff
@@ -225,8 +226,8 @@ GEM
     matrix (0.4.2)
     method_source (1.0.0)
     mini_mime (1.1.2)
-    minitest (5.15.0)
-    msgpack (1.4.5)
+    minitest (5.16.3)
+    msgpack (1.5.6)
```

Comment:
### [msgpack](http://msgpack.org/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.4.5 → 1.5.6

[_~~change-log~~, [source-code](https://github.com/msgpack/msgpack-ruby), [gem-diff](https://my.diffend.io/gems/msgpack/1.4.5/1.5.6)_]





<details>
<summary>Commits</summary>

A change of **58** commits. See the full changes on [the compare page](https://github.com/msgpack/msgpack-ruby/compare/v1.4.5...v1.5.6).

These are the first 10 commits:
- (630a933) [delete all macro definitions for compat](https://github.com/msgpack/msgpack-ruby/commit/630a9330a75f3ab4e8503c0212ca1688a4b9da05)
- (42fb525) [Remove RUBINIUS specific code](https://github.com/msgpack/msgpack-ruby/commit/42fb5256a275b4798e54d9c5dd0224444d1bbfcf)
- (e374321) [Exclude time edge cases on TruffleRuby as the full epoch is …](https://github.com/msgpack/msgpack-ruby/commit/e374321a6986859c3e896e2d159b4ec44be70cfc)
- (cc84bf3) [Optimize Packer and Unpacker initialize methods](https://github.com/msgpack/msgpack-ruby/commit/cc84bf3449722bdff5a3761ba20f10c86a6bac0e)
- (b9ca04a) [Merge pull request #253 from Shopify/opti-messagepack-load](https://github.com/msgpack/msgpack-ruby/commit/b9ca04a139bbd82762f9982af6fa78f0516bbec7)
- (84ca67c) [Support serializing bigint as extension types](https://github.com/msgpack/msgpack-ruby/commit/84ca67c02b3102e6db19262dbc53bc453a0c4f5e)
- (9aa71da) [Merge pull request #244 from Shopify/max-big-num](https://github.com/msgpack/msgpack-ruby/commit/9aa71da1bb4f61a5b4a6291522ef90aff9d96b58)
- (5f9e9f4) [Implement Factory#pool](https://github.com/msgpack/msgpack-ruby/commit/5f9e9f4f1e65ab558ae743905164d87e02948a51)
- (04ccbd8) [Merge pull request #266 from Shopify/factory-pool](https://github.com/msgpack/msgpack-ruby/commit/04ccbd8e11c894cbf235b497b2dbd93338a647b8)
- (0298897) [Remove MessagePack::Bigint::TYPE constant](https://github.com/msgpack/msgpack-ruby/commit/0298897e664c1f0e97a0dfa5e14604f3ad0b1cd0)


</details>